test/spec/app_spec.rb

```diff
@@ -55,6 +53,7 @@
 
   before :each do |example|
     $caller_name = example.metadata[:description_args]
+    p $caller_name
```

Comment:
Remove debugging output

Reply:
Until the end of the day, when finishing a stable version

Reply:
Removed